Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -23,6 +23,7 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
         initializer: function () {
             this.on('*:logOut', this._logOut);
             this.after('*:navigateTo', this._navigateTo);
+            this.after('*:activeNavigationChange', this._activeNavigationChange);
```

Comment:
`_activeNavigationChange` is a very bad name as it gives absolutely no clue of it is doing. Good method names usually start with a verb and tell what the method does, not why/when it is called. So here for instance, the name could be `_navigateToFirstItemRoute`. That way, you can almost read that line as an English sentence.